Emscripten-build-instructions.md

```diff
@@ -0,0 +1,167 @@
+# Wasm Build Instructions
+
+## CppInterOp Wasm Build Instructions
+
+This document first starts with the instructions on how to build a wasm  
+build of CppInterOp. Before we start it should be noted that  
+unlike the non wasm version of CppInterOp we currently only  
+support the Clang-REPL backend using llvm>19 for osx and Linux.  
+We will first make folder to build our wasm build of CppInterOp.  
+This can be done by executing the following command
+
+```bash
+mkdir CppInterOp-wasm
+```
+
+Now move into this directory using the following command
+
+```bash
+cd ./CppInterOp-wasm
+```
```

Comment:
Name for a build directory is weird. Just use build.

Reply:
Also order looks weird 

1) setup emsdk first
2) any cloning that's required
3) create the build directory
4) The build configs using cmake
5) make step

Reply:
This is a top levell build directory to keep everything contained if the development build goes wrong. For those who simple want to experiment with it then I shalll point then tk the emscripten forge package